### PR TITLE
update commands and tool info about `key-type`

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Updated two commands and the tool info needed for modifying `key-type` regarding the feedback from AskTUG, https://asktug.com/t/topic/213152.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: best-practices/pd-scheduling-best-practices.md [hunk 1]
```diff
@@ -264,14 +264,30 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
-        + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
-        + 或者 `key-type` 保持 `table`，同时设置 `enable-cross-table-merge`为 `true`，该参数支持动态修改。
-       
-        > **注意：**
-        >
-        > 在开启 `placement-rules`后，请合理切换 `txn`和 `raw`，避免无法正常解码 key。
+  - TiKV: 将 `split-region-on-table` 设为 `false`，该参数不支持动态修改。
+  - PD: 使用 PD Control，选择性地设置以下参数。
+
+    * 如果集群中不存在 TiDB 实例，[`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值应该被设置为 "raw" 或 "txn"。此时，无论 enable-cross-table-merge 设置为何，PD 均可以跨表合并 Region。 该参数支持动态修改。
```
Comment (lhy1024) [inline best-practices/pd-scheduling-best-practices.md:270]:
```suggestion
    * 如果集群中不存在 TiDB 实例，[`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值应该被设置为 `raw` 或 `txn`。此时，无论 enable-cross-table-merge 设置为何，PD 均可以跨表合并 Region。 该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 2]
```diff
@@ -264,14 +264,30 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
-        + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
-        + 或者 `key-type` 保持 `table`，同时设置 `enable-cross-table-merge`为 `true`，该参数支持动态修改。
-       
-        > **注意：**
-        >
-        > 在开启 `placement-rules`后，请合理切换 `txn`和 `raw`，避免无法正常解码 key。
+  - TiKV: 将 `split-region-on-table` 设为 `false`，该参数不支持动态修改。
+  - PD: 使用 PD Control，选择性地设置以下参数。
+
+    * 如果集群中不存在 TiDB 实例，[`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值应该被设置为 `raw` 或 `txn`。此时，无论 enable-cross-table-merge 设置为何，PD 均可以跨表合并 Region。 该参数支持动态修改。
```
Comment (en-jin19) [inline best-practices/pd-scheduling-best-practices.md:270]:
```suggestion
  - PD: 使用 PD Control，根据集群情况选择性地设置以下参数。

    * 如果集群中不存在 TiDB 实例，应该将 [`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值设置为 `raw` 或 `txn`。此时，无论 `enable-cross-table-merge` 设置如何，PD 均可以跨表合并 Region。 该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 3]
```diff
@@ -264,14 +264,30 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
+     {{< copyable "shell-regular" >}}
+
+        ```bash
+         config set key-type txn
+        ```
+
+    * 如果集群中存在 TiDB 实例，`key-type` 应该为 `table`。此时设置 `enable-cross-table-merge` 为 `true`，可以让 PD 能跨表合并 Region。该参数支持动态修改。
```
Comment (en-jin19) [inline best-practices/pd-scheduling-best-practices.md:278]:
```suggestion
    * 如果集群中存在 TiDB 实例，`key-type` 的值应该设置为 `table`。此时设置 `enable-cross-table-merge` 为 `true`，可以让 PD 跨表合并 Region。该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md (hunk 2 above)
Comment (lhy1024) [reply best-practices/pd-scheduling-best-practices.md:270]: Good

File: best-practices/pd-scheduling-best-practices.md (hunk 3 above)
Comment (lhy1024) [reply best-practices/pd-scheduling-best-practices.md:278]: Good

**Review by lhy1024 — APPROVED** [review]

File: best-practices/pd-scheduling-best-practices.md [hunk 4]
```diff
@@ -264,14 +264,30 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
 
-    - TiKV: 将 `split-region-on-table` 设为 `false`，该参数不支持动态修改。
-    - PD: 
-        + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
-        + 或者 `key-type` 保持 `table`，同时设置 `enable-cross-table-merge`为 `true`，该参数支持动态修改。
-       
-        > **注意：**
-        >
-        > 在开启 `placement-rules`后，请合理切换 `txn`和 `raw`，避免无法正常解码 key。
+  - TiKV: 将 `split-region-on-table` 设为 `false`，该参数不支持动态修改。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:267]: 相邻两级的缩进都为四个空格，请修改缩进空格。

Comment (en-jin19) [reply best-practices/pd-scheduling-best-practices.md]: 已修改，PTAL~

File: best-practices/pd-scheduling-best-practices.md [hunk 5]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
 - 创建过大量表后（包括执行 `Truncate Table` 操作）又清空了。此时如果开启了 split table 特性，这些空 Region 是无法合并的，此时需要调整以下参数关闭这个特性：
 
     - TiKV: 将 `split-region-on-table` 设为 `false`，该参数不支持动态修改。
-    - PD: 
-        + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
-        + 或者 `key-type` 保持 `table`，同时设置 `enable-cross-table-merge`为 `true`，该参数支持动态修改。
-       
+    - PD: 使用 PD Control，根据集群情况选择性地设置以下参数。
+
+        * 如果集群中不存在 TiDB 实例，应该将 [`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值设置为 `raw` 或 `txn`。此时，无论 `enable-cross-table-merge` 设置如何，PD 均可以跨表合并 Region。 该参数支持动态修改。
```
Comment (en-jin19) [inline best-practices/pd-scheduling-best-practices.md:270]:
```suggestion
        * 如果集群中不存在 TiDB 实例，应该将 [`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值设置为 `raw` 或 `txn`。此时，无论 `enable-cross-table-merge` 设置如何，PD 均可以跨表合并 Region。 该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 6]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
+        * 如果集群中存在 TiDB 实例，`key-type` 的值应该设置为 `table`。此时设置 `enable-cross-table-merge` 为 `true`，可以让 PD 跨表合并 Region。该参数支持动态修改。
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        config set enable-cross-table-merge true
+        ```
+
+        如果遇到修改未生效的问题，请参阅[修改 TiKV/PD 的 toml 配置文件后没有生效](/deploy-and-maintain-faq.md#为什么修改了-tikvpd-的-toml-配置文件却没有生效)。
```
Comment (en-jin19) [inline best-practices/pd-scheduling-best-practices.md:286]:
```suggestion
        如果遇到修改未生效的问题，请参阅[修改 TiKV/PD 的 toml 配置文件后没有生效](/faq/deploy-and-maintain-faq.md#为什么修改了-tikvpd-的-toml-配置文件却没有生效)。
```

File: best-practices/pd-scheduling-best-practices.md (hunk 5 above)
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:270]:
```suggestion
        * 如果集群中不存在 TiDB 实例，将 [`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值设置为 `raw` 或 `txn`。此时，无论 `enable-cross-table-merge` 设置如何，PD 均可以跨表合并 Region。该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 7]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
+        * 如果集群中不存在 TiDB 实例，应该将 [`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值设置为 `raw` 或 `txn`。此时，无论 `enable-cross-table-merge` 设置如何，PD 均可以跨表合并 Region。 该参数支持动态修改。
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        config set key-type txn
+        ```
+
+        * 如果集群中存在 TiDB 实例，`key-type` 的值应该设置为 `table`。此时设置 `enable-cross-table-merge` 为 `true`，可以让 PD 跨表合并 Region。该参数支持动态修改。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:278]:
```suggestion
        * 如果集群中存在 TiDB 实例，应该将 `key-type` 的值设置为 `table`。此时将 `enable-cross-table-merge` 设置为 `true`，可以使 PD 跨表合并 Region。该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 8]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        config set enable-cross-table-merge true
+        ```
+
+        如果遇到修改未生效的问题，请参阅[修改 TiKV/PD 的 toml 配置文件后没有生效](/faq/deploy-and-maintain-faq.md#为什么修改了-tikvpd-的-toml-配置文件却没有生效)。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:286]:
```suggestion
        如果修改未生效，请参阅[ FAQ - 修改 TiKV/PD 的 toml 配置文件后没有生效](/faq/deploy-and-maintain-faq.md#为什么修改了-tikvpd-的-toml-配置文件却没有生效)。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 9]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
         > **注意：**
         >
-        > 在开启 `placement-rules`后，请合理切换 `txn`和 `raw`，避免无法正常解码 key。
+        > 在开启 `placement-rules`后，请合理切换 `key-type`，避免无法正常解码 key。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:290]:
```suggestion
        > 在开启 `placement-rules` 后，请合理切换 `key-type`，避免无法正常解码 key。
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

File: best-practices/pd-scheduling-best-practices.md (hunk 5 above)
Comment (en-jin19) [reply best-practices/pd-scheduling-best-practices.md:270]: @TomShawn 这里可否保留 ”应该将“，做到和下一个模块统一呢？

Comment (TomShawn) [reply best-practices/pd-scheduling-best-practices.md:270]: ”应该“在技术写作中或是在这个上下文中，  都是一个奇怪的措辞，你觉得呢？

Comment (en-jin19) [reply best-practices/pd-scheduling-best-practices.md:270]: 那么我们把两个 ”应该将“ 都替换为 ”将“ 如何？确实只使用 ”将“ 会更明确些

File: best-practices/pd-scheduling-best-practices.md [hunk 10]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
+        * 如果集群中不存在 TiDB 实例，应该将 [`key-type`](/pd-control.md#config-show--set-option-value--placement-rules) 的值设置为 `raw` 或 `txn`。此时，无论 `enable-cross-table-merge` 设置如何，PD 均可以跨表合并 Region。 该参数支持动态修改。
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        config set key-type txn
+        ```
+
+        * 如果集群中存在 TiDB 实例，应该将 `key-type` 的值设置为 `table`。此时将 `enable-cross-table-merge` 设置为 `true`，可以使 PD 跨表合并 Region。该参数支持动态修改。
```
Comment (en-jin19) [inline best-practices/pd-scheduling-best-practices.md:278]:
```suggestion
        * 如果集群中存在 TiDB 实例，将 `key-type` 的值设置为 `table`。此时将 `enable-cross-table-merge` 设置为 `true`，可以使 PD 跨表合并 Region。该参数支持动态修改。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: best-practices/pd-scheduling-best-practices.md [hunk 11]
```diff
@@ -265,13 +265,29 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+
+        * 如果集群中存在 TiDB 实例，将 `key-type` 的值设置为 `table`。此时将 `enable-cross-table-merge` 设置为 `true`，可以使 PD 跨表合并 Region。该参数支持动态修改。
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        config set enable-cross-table-merge true
+        ```
+
+        如果修改未生效，请参阅[ FAQ - 修改 TiKV/PD 的 toml 配置文件后没有生效](/faq/deploy-and-maintain-faq.md#为什么修改了-tikvpd-的-toml-配置文件却没有生效)。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:286]:
```suggestion
        如果修改未生效，请参阅 [FAQ - 修改 TiKV/PD 的 toml 配置文件后没有生效](/faq/deploy-and-maintain-faq.md#为什么修改了-tikvpd-的-toml-配置文件却没有生效)。
```

**Pull request merged**